solution/backend/file_manager/views.py

```diff
@@ -76,6 +99,23 @@ def get_queryset(self):
             Prefetch("locations", queryset=locations_prefetch),
             Prefetch("subject", queryset=subjects_prefetch),
             Prefetch("document_type", queryset=doc_type_prefetch)).distinct()
+
+        if search_query:
+            (search_type, cover_density) = (
+                ("phrase", True)
+                if search_query.startswith('"') and search_query.endswith('"')
+                else ("plain", False)
+            )
+            query = query.annotate(
+                rank=SearchRank(
+                    SearchVector("name", "file_name", "date", "description", weight="A", config="english"),
```

Comment:
I would remove date and file name from the search vector.  Date is way to inconsistent of an input to properly do a search on and file_name is more or less a meta data type of object.  "Untitled" can be the name of the file that is uploaded.  It most likely wouldnt have any significance in search

Reply:
Also put name and description as two separate ranks, A & B.   it might give us an easier time to play around with ranks if they are separate weights for now.

Reply:
Asked Britta about this and she suggested that date and filename search could be helpful for now until we have a better idea what we will do with file names. Will do ranks.